helm/superset/values.yaml

```diff
@@ -200,6 +200,8 @@ service:
     {}
     # cloud.google.com/load-balancer-type: "Internal"
   loadBalancerIP: null
+  nodePorts:
```

Comment:
Since the `service.port` is already singular, let's just make this `nodePort`

Reply:
Updated, thanks!